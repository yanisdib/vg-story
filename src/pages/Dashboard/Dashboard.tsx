import { Outlet } from 'react-router-dom';

import {
    Content,
    View,
    Wrapper
} from './Dashboard.styles';


function Dashboard(): JSX.Element {
    
    return (
        <View>
            <Wrapper>
                <Content>
                    Dashboard
                    <Outlet />
                </Content>
            </Wrapper>
        </View>
    );
}


export default Dashboard;