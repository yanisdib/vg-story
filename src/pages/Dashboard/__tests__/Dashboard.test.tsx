import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

import Dashboard from '../Dashboard';


describe('Dashboard page', () => {
    it('should render without crashing', () => {

        const { container } = render(<Dashboard />, { wrapper: MemoryRouter });

        expect(container).toMatchSnapshot();
    });
});