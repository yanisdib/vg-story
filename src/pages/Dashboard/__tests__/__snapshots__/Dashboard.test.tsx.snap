// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard page should render without crashing 1`] = `
<div>
  <div
    class="sc-bczRLJ"
  >
    <div
      class="sc-gsnTZi"
    >
      <div
        class="sc-dkzDqf"
      >
        Dashboard
      </div>
    </div>
  </div>
</div>
`;
